staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -989,6 +989,8 @@ const (
 	HypervLabel = "hyperv.node.kubevirt.io/"
 	// This label represents vendor of cpu model on the node
 	CPUModelVendorLabel = "cpu-vendor.node.kubevirt.io/"
+	// This label represents supported machine type on the node
+	SupportedMachineTypeLabel = "machine-type.node.kubevirt.io/"
```

Comment:
This is the only "api change" here, hence the approval.

the api-change label would be gone if we'd get rid of this const and use the label by-value, but I don't find that beneficial in any way.

@enp0s3 WDYT?

Reply:
In addition, there are other examples for backports of similar "api changes" [[1](https://github.com/kubevirt/kubevirt/pull/13976/files#diff-34a591e3aa47260cfad791064c20f34f98fdca4a63532fbf7184e68682f4fcb8L2525)][[2](https://github.com/kubevirt/kubevirt/pull/14421/files#diff-df03d2826ecf8b2a2c21bbfec2ec08685fbb7ce0486696ccbfdd074f5e50c82cL204)].

Reply:
Now that I am thinking of it, I don't see a reason to host these consts in our API, since these consts are not related to the user-face API, they are not used by our CRs. Perhaps we should maintain them in the node labeller package.

Reply:
+1 that this in not API change and good to be merged